client/src/components/layout/NewDogForm.js

```diff
@@ -0,0 +1,122 @@
+import React, { useState } from "react"
+import ErrorList from "./ErrorList.js"
+import { Redirect } from "react-router-dom";
+import Fetcher from "./services/Fetcher.js";
+
+const NewDogForm = ({ user }) => {
+  const defaultFormValue = {
+    dogName: "",
+    pictureUrl: "",
+    description: "",
+    userId: user.id,
```

Comment:
The express side of the application is already keeping track of the signed in user through the passport session - in the router we can access this through `req.user`, so React here doesn't need to send the user with the request, it should just be the body of data